Split Ads how-to guides into per-platform guides (#772)
THEOplayer/documentation: theoplayer/how-to-guides/ios/ads/google-ima.mdx

---
sidebar_position: 10
sidebar_label: Google IMA
---

# Google IMA on iOS

import Intro from '../../shared/ads/_google-ima-intro.mdx';

<Intro />

To use Google IMA with the THEOplayer iOS SDK, the **THEOplayer GoogleIMA Integration** should be integrated. The integration is a lightweight module written in Swift, for serving advertisements from the Google IMA SDK.

:::info
The THEOplayer GoogleIMA integration supports both iOS and tvOS platforms.
:::

## Installation

The THEOplayer GoogleIMA integration is published on the following package managers:

### Cocoapods

Simply add the following to your project's Podfile:

```ruby
pod 'THEOplayer-Integration-GoogleIMA'
```

The above entry automatically manages the fetching of the IMA SDK dependency.

> In case where the integrator would want to manage fetching a specific version of the IMA SDK, then replace the Podfile entry with:

Reviewer: ```suggestion
If you would want to set a specific version for the IMA SDK, replace the Podfile entry with:
```

```ruby
pod 'THEOplayer-Integration-GoogleIMA/Base'
pod 'GoogleAds-IMA-iOS-SDK', '3.31.0' # specify the desired version
# or
pod 'GoogleAds-IMA-tvOS-SDK', '4.16.0' # specify the desired version
```

### Swift Package Manager

> Please check the [installation instruction found here](https://github.com/THEOplayer/theoplayer-sdk-apple/README.md#installation)

Reviewer: ```suggestion
Please check the [installation instruction found here](https://github.com/THEOplayer/theoplayer-sdk-apple/README.md#installation).
```
